research/templates/research/metadata-form.html

```diff
@@ -75,7 +75,7 @@ <h5 class="card-title float-start">
 <div id="metadata-form" class="row hide">
     <div class="col-md-12">
         <div class="card">
-            <div class="card-header">
+            <div class="card-header sticky-top metadata-card">

```

Comment:
Hi,
The banner is only for non-production environments. In production environment, it will look like this.

<img width="1316" alt="Screenshot 2024-09-25 at 13 37 41" src="https://github.com/user-attachments/assets/b23b5cad-798a-42bc-a7b2-92286045f3dd">

Reply:
Added padding to metadata form header if maintenance banner exists.